Sources/FLCharts/FLCartesianPlane.swift

```diff
@@ -387,6 +395,10 @@ public class FLCartesianPlane: UIView, FLStylable {
         case .right:
             leadingConstant = showAverageLine ? marginForAverageView + margin.left : margin.left
             trailingConstant = margin.right + config.axesLines.lineWidth
+                
+        case .none:
+            leadingConstant = showAverageLine ? marginForAverageView + margin.left : margin.left
```

Comment:
@SvenMuc Thank you for your improvement!

I think that this line should be 'leadingConstant = margin.left', because the average line will be displayed on the right of the chart.